### DISCUSSION TITLE
Exe file does not work in different computer

### DISCUSSION THREAD

**Discussion by khanalim** [brentvollebregt/auto-py-to-exe#332] [Q&A]
Hello,
I can create exe file from python script in my Mac using "auto-py-to-exe" and can run it without any issues. However, when I move the exe file to another computer regardless of Mac or Windows, it does not work. The new computer even does not recognize the exe file as exe. 
Here are some useful information about my python script. I have used Numpy, Pandas, matplotlib, xarray, math libraries. I also need an input for program to run. So, I choose "one folder" option while auto-py-to-exe runs, so that I can change the input file later.
Any suggestions?

**Comment by khanalim** [comment]
I like to add few more information. I realized the created exe file is considered as security risk by other computers. Not sure how to make is safe.

**Comment by brentvollebregt** [comment]
See my comment at https://github.com/brentvollebregt/auto-py-to-exe/discussions/334#discussioncomment-4262353
